load-jetpack.php

```diff
@@ -100,5 +96,3 @@ function jetpack_should_use_minified_assets() {
 }
 
 require_once JETPACK__PLUGIN_DIR . '3rd-party/3rd-party.php';
-
-Jetpack::init();
```

Comment:
Do we need to remove `Jetpack::init()` here? 

Since we want to move away from using `Jetpack::init()` in other classes and instead use the `jetpack_loaded` action to obtain the Jetpack object, we'll need to call `Jetpack::init()` somewhere to create the Jetpack object.

Reply:
Added it back, thank you!